More osu! patterns and [PL] updates for patterns (#11983)
ppy/osu-wiki: wiki/Beatmap/Pattern/osu!/Bursts/en.md

---
stub: true
tags:
  - bursts
---

# Bursts

> **Bursts** are a higly used pattern in [beatmaps](/wiki/Beatmap). They consist of 3 or more [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) that are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on eachother.

Reviewer: ```suggestion
**Bursts** consist of three or more [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) that are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on top of each other.
```
once again, not any more or so clearer but a step in the right direction

> Bursts mostly consist of 1/4 or faster rythms.

Reviewer: "Bursts are a highly used pattern in beatmaps" Even so, starting it off like this instead of describing what a burst seems redundant, not saying I disagree.

```suggestion
Bursts are commonly snapped to 1/4 ticks.
```
